machines/sim_filelist.mk

```diff
@@ -4,6 +4,7 @@
 VHEADERS += $(BSG_MANYCORE_DIR)/testbenches/common/v/bsg_manycore_mem_cfg_pkg.v
 VHEADERS += $(BSG_MANYCORE_DIR)/testbenches/common/v/bsg_manycore_network_cfg_pkg.v
 VHEADERS += $(BSG_MANYCORE_DIR)/testbenches/common/v/bsg_manycore_profile_pkg.v
+VHEADERS += $(BASEJUMP_STL_DIR)/bsg_dmc/bsg_dmc_pkg.v
```

Comment:
This will be needed for future basejump master based on https://github.com/bespoke-silicon-group/basejump_stl/pull/417. Can remove if needed, but doesn't hurt

Reply:
I would just remove it because there is no plan on maintaining dmc backend

Reply:
What do you mean by this, tommy?

Reply:
@tommydcjung

Reply:
Current testbench does not have dmc as a memory backend.

Reply:
Sure but we do plan to maintain the dmc backend. :-)

Reply:
Removed for now